__init__.py

```diff
@@ -191,7 +192,10 @@ def _start_new_timer(self, message):
         else:
             timer = self._build_timer(duration, name)
             self.active_timers.append(timer)
-            self.active_timers.sort(key=lambda x: x.expiration)
+            self.active_timers.sort(key=lambda tmr: tmr.expiration)
+            if len(self.active_timers) == 1:
+                self._show_gui()
+                self._start_display_update()
             self._speak_new_timer(timer)
             self._save_timers()
 
```

Comment:
In communicate_timer_status() is no timer matches you do nothing? Shouldn't you say something like timer not found in this case?

Reply:
if you look at the logic in `_speak_timer_status_matches`, a `timer-not-found` dialog is spoken if there is no match.